cluster-state-service/handler/httpclient/httpclient.go

```diff
@@ -47,10 +51,11 @@ func init() {
 
 // New returns an Blox httpClient that will insert custom HTTP UA header.
 func New() *http.Client {
-	transport := &http.Transport{}
+	transport := http.DefaultTransport
 
 	client := &http.Client{
 		Transport: &bloxRoundTripper{transport},
+		Timeout: httpClientTimeout,
```

Comment:
Out of curiosity, what is the different between this and the timeout in http.DefaultTransport?